cmake/FindIgnOGRE.cmake

```
@@ -83,6 +83,16 @@ if (NOT WIN32)
 
   # loop through pkg config paths and find an ogre version that is < 2.0.0
   foreach(pkg_path ${PKG_CONFIG_PATH_TMP})
+
+    if (NOT EXISTS ${pkg_path})
```

```suggestion
  foreach(pkg_path ${PKG_CONFIG_PATH_TMP})
    if (NOT EXISTS ${pkg_path})
```
nit: remove whitespace?

---

3969bcc